pandas/util/testing.py

```diff
@@ -1345,10 +1348,10 @@ def assert_frame_equal(left, right, check_dtype=True,
     check_like : bool, default False
         If True, ignore the order of index & columns.
         Note: index labels must match their respective rows
-        (same as in columns) - same labels must be with the same data
+        (same as in columns) - same labels must be with the same data.
     obj : str, default 'DataFrame'
         Specify object name being compared, internally used to show appropriate
-        assertion message
+        assertion message.
     """
```

Comment:
I think it's worth adding a `See Also` section, with `DataFrame.equals`, `testing.assert_series_equal` and may be others.

Reply:
Added